README.md

```diff
@@ -3,6 +3,14 @@
 Provides an interface between [Flask](http://flask.pocoo.org/) web framework
 and [humanize](https://github.com/jmoiron/humanize) library.
 
+## Configuration
```

Comment:
Move content of "Configuration" section into "Features" for consistency with other features.